examples/bmp3xx_simpletest/bmp3xx_simpletest.ino

```diff
@@ -41,11 +41,11 @@ void setup() {
     while (1);
   }
 
-  // Set up oversampling and filter initialization
-  bmp.setTemperatureOversampling(BMP3_OVERSAMPLING_8X);
-  bmp.setPressureOversampling(BMP3_OVERSAMPLING_4X);
-  bmp.setIIRFilterCoeff(BMP3_IIR_FILTER_COEFF_3);
-  //bmp.setOutputDataRate(BMP3_ODR_50_HZ);
+  // for details see chapter 3.5 Table 9
```

Comment:
dont remove the comment where we explain it. also fix your whitespace here

Reply:
also say 'where' the chapter is, is it the datasheet? make that explicit. people can't read your mind